tests/acceptance/view-tree-test.js

```diff
@@ -186,6 +187,71 @@ test("It should correctly display the view tree", async function(assert) {
   ], 'expected title tips');
 });
 
+test("It should filter the view tree using the search text", async function(assert) {
+  let viewTree = defaultViewTree();
+
+  await visit('/');
+  run(() => {
+    port.trigger('view:viewTree', { tree: viewTree });
+  });
+  await wait();
+
+  let treeNodes = findAll('.js-view-tree-item');
+  assert.equal(treeNodes.length, 3, 'expected some tree nodes');
+
+  fillIn('.js-filter-views input', 'post');
```

Comment:
`await`

Reply:
I didn't realize we had to use `await` for async helpers because I thought the test framework automatically took care of making it all synchronous. I read the docs again, and is it that the awareness comes from preceding async tasks, rather than ensuring downstream logic is synchronous? (I hope I said that clearly...)

In other words:
```
asyncFunction();
// Async helper is aware of the above async behaviour and will wait
fillIn();
// But it doesn't make sure the stuff below gets executed in a synchronous way
some = synchronousLogic(); 
```
So to fix it, I would have to use either `await` or `andThen`?
```
await fillIn();
// or
andThen(() => { some = synchronousLogic() });
```

Reply:
Yes precisely, you got it. `await` is used to avoid using `andThen` down the line.